src/pyfme/aircrafts/cessna_172.py

```diff
@@ -241,13 +248,13 @@ def _calculate_aero_lon_forces_moments_coeffs(self):
         )
         # FIXME: CM_q multiplicado por 2 hasta que alpha_dot pueda ser calculado
 
-    def _calculate_aero_lat_forces_moments_coeffs(self):
+    def _calculate_aero_lat_forces_moments_coeffs(self, system):
         delta_aile = np.rad2deg(self.controls['delta_aileron'])  # deg
         delta_rud_RAD = self.controls['delta_rudder']  # rad
         alpha_DEG = np.rad2deg(self.alpha)  # deg
         b = self.span
         V = self.TAS
-        p, q, r = self.p, self.q, self.r
+        p, q, r = system.p, system.q, system.r

```

Comment:
So now the velocities belong to the `System`, rather than the `Aircraft`. Is that correct?